src/installer/corehost/cli/exe.cmake

```diff
@@ -16,10 +17,10 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/fxr)
 list(APPEND SOURCES
     ${CMAKE_CURRENT_LIST_DIR}/fxr_resolver.cpp
     ${CMAKE_CURRENT_LIST_DIR}/../corehost.cpp
+    ${HOSTMISC_SRC}
```

Comment:
Should this file also add `HOSTMISC_HDR` to `HEADERS`?

Reply:
No longer needed.